iolog.h

```diff
@@ -41,7 +41,8 @@ union io_sample_data {
 struct io_sample {
 	uint64_t time;
 	union io_sample_data data;
-	uint32_t __ddir;
+	uint32_t __ddir :31;
```

Comment:
Don't like this at all! Instead of playing games with this, just add a separate field. There's a 4 byte hole between __ddir and bs anyway.